### PR TITLE
Style/main page

### LINKED ISSUE #77: Make MainPage layout as in sketches

### PR DESCRIPTION
**Pull request by ingeridhellen**
## Why is this pull request needed?

This pull request is needed because we want our main page to look good

## What does this pull request change?

Aligns main page to sketches. Since some features are missing, not everything is perfect yet, but this gets us at least halfway there

## Issues related to this change:

Closes #77

### CODE REVIEW

File: web/src/components/feature/PhaseTable.tsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const PhaseTable = (props: {
         'Mercury Flow (g/d)',
       ]}
       rows={getRows(multiFlashResponse, cubicFeedFlow)}
+      density={'comfortable'}
```
Comment (mariuswinger) [inline web/src/components/feature/PhaseTable.tsx:34]: Do we want the tables to have different density?

File: web/src/pages/Main.tsx [hunk 2]
```diff
@@ -8,6 +10,26 @@ import { ComponentResponse, MultiflashResponse } from '../api/generated'
 import { ComponentSelector } from '../components/feature/ComponentSelector'
 import { PhaseTable } from '../components/feature/PhaseTable'
 
+const Results = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  gap: 50px;
+  align-items: center;
+  flex-wrap: wrap;
```
Comment (mariuswinger) [inline web/src/pages/Main.tsx:18]: This behaves a bit weird when resizing the window (i.e. dev tools). It wraps when the window becomes to small to have both tables on the same row as expected, but when expanding it puts them on the same row until some point where it wraps, even though there is enough space, before putting them on the same row again.
I think it has something to do with min-width: 1200px in Container.

**Review by mariuswinger — APPROVED** [review]

**Comment by mariuswinger** [comment]
Would be nice to see this with a rebase so the tables are generated when clicking run.

File: web/src/components/feature/PhaseTable.tsx (hunk 1 above)
Comment (ingeridhellen) [reply web/src/components/feature/PhaseTable.tsx:34]: According to the sketches they should

**Pull request merged**